Server/ReasnAPI/ReasnAPI.Tests/Services/ObjectTypeServiceTests.cs

```diff
@@ -0,0 +1,200 @@
+﻿using ReasnAPI.Models.Database;
+using ReasnAPI.Models.DTOs;
+using ReasnAPI.Services;
+using Moq;
+using Moq.EntityFrameworkCore;
+
+namespace ReasnAPI.Tests.Services {
+    [TestClass]
+    public class ObjectTypeServiceTests {
+        [TestMethod]
+        public void GetObjectTypeById_ObjectTypeExist_ObjectTypeReturned() {
+            var mockContext = new Mock<ReasnContext>();
+            mockContext.Setup(c => c.ObjectTypes).ReturnsDbSet([
+                new() { Id = 1, Name = "Type1" }

```

Comment:
Remove redundant "()".

Reply:
It is not possible to do that here because it causes an error